cmd/subnetcmd/create.go

```diff
@@ -87,7 +88,9 @@ configuration, pass the -f flag.`,
 	cmd.Flags().BoolVar(&createFlags.useLatestReleasedVMVersion, latest, false, "use latest Subnet-EVM released version, takes precedence over --vm-version")
 	cmd.Flags().Uint64Var(&createFlags.chainID, "evm-chain-id", 0, "chain ID to use with Subnet-EVM")
 	cmd.Flags().StringVar(&createFlags.tokenSymbol, "evm-token", "", "token symbol to use with Subnet-EVM")
-	cmd.Flags().BoolVar(&createFlags.useDefaults, "evm-defaults", false, "use default settings for fees/airdrop/precompiles/teleporter with Subnet-EVM")
+	cmd.Flags().BoolVar(&createFlags.useProductionDefaults, "evm-defaults", false, "use default production settings with Subnet-EVM")
```

Comment:
What's the difference between evm-defaults and production-defaults?

Reply:
no difference. this is just backwards compatibility. evm-defaults has been present for a long time. will mark it as a deprecation notice.